### PR TITLE
chore: Update showcase to latest version

### PR DESCRIPTION
**Pull request by lqiu96**
Thank you for opening a Pull Request! For general contributing guidelines, please refer to [contributing guide](https://github.com/googleapis/gapic-generator-java/blob/main/CONTRIBUTING.md)

Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

- [x] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/gapic-generator-java/issues/new/choose) before writing your code! That way we can discuss the change, evaluate designs, and agree on the general idea
- [x] Ensure the tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)

Update the latest version to include this fix: https://github.com/googleapis/gapic-showcase/pull/1282

### CODE REVIEW

**Review by mpeddada1 — APPROVED** [review]
Could this be automated through renovate-bot? (https://docs.renovatebot.com/modules/datasource/github-releases/)

**Comment by lqiu96** [comment]
> Could this be automated through renovate-bot? (https://docs.renovatebot.com/modules/datasource/github-releases/) cc/ @burkedavison

Oh, that's cool! Let me try that before merging this in (add in a temp `Do not merge`). I was thinking about making a GH action in the future, but this might be easier.

**Pull request merged**
